packages/grid/_modules_/grid/hooks/features/columns/useGridColumns.ts

```diff
@@ -196,6 +196,16 @@ export function useGridColumns(columns: GridColumns, apiRef: GridApiRef): void {
     [apiRef, gridState.columns, logger, updateState],
   );
 
+  const setColumnWidth = React.useCallback(
+    (field: string, width: number) => {
+      logger.debug(`Updating column ${field} width to ${width}`);
+
+      const column = apiRef.current.getColumnFromField(field);
+      updateColumn({ ...column, width });
```

Comment:
You should publish the event after that
`apiRef.current.publishEvent(COLUMNSIZE_CHANGED...)`

Reply:
Done. The event is `GRID_COLUMN_RESIZE_COMMITED`